src/components/agent-details/AgentListings/agent-listings.style.js

```diff
@@ -0,0 +1,131 @@
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
+import styled from 'styled-components';
+
+export const ListingsContainer = styled.div`
+  margin: 0rem 1rem;
+
+  ${mq.desktop`
+    margin: 0rem 3.4375rem;
+  `};
+`;
+
+export const CardsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+
+  ${mq.desktop`
+    flex-direction:row;
+  `};
+`;
+
+export const Card = styled.div`
+  box-shadow: 0rem 0.0625rem 0.375rem ${colors.mineShaft}26;
+`;
+
+export const CardImageContainer = styled.div`
+  position: relative;
+`;
+
+export const Image = styled.img`
+  width: 100%;
+`;
+
+export const CornerItems = styled.div`
+  display: flex;
+  position: absolute;
+  bottom: ${(props) => (props.bottom ? '1rem' : null)};
+  left: ${(props) => (props.left ? '1rem' : null)};
+  right: ${(props) => (props.right ? '1rem' : null)};
+  top: ${(props) => (props.top ? '1rem' : null)};
```

Comment:
This can be enhanced
```suggestion
  ${({ isBottom }) => (isBottom ? `bottom: 1rem;` : `top: 1rem;`)}
  ${({ isRight }) => (isRight ? `right: 1rem;` : `left: 1rem;`)}
```